client/src/main/java/com/distkv/dst/client/CheckStatusUtil.java

```diff
@@ -24,7 +24,7 @@ public static void checkStatus(CommonProtocol.Status status, String key, String
       case SLIST_MEMBER_NOT_FOUND:
         throw new SortedListMemberNotFoundException(key, typeCode);
       case SLIST_TOPNUM_BE_POSITIVE:
-        throw new SortedListTopNumBePositiveException(key, typeCode);
+        throw new SortedListTopNumBeNonNegativeException(key, typeCode);
```

Comment:
I don't `SortedListTopNumBeNonNegativeException ` is a good exception here.
Maybe `NonNegativeException` will be more general.

Reply:
I think no more generality needed here.

Reply:
If we need to redesign exception, we need to repush new PR.

Reply:
I would rename **SortedListTopNumBeNonNegativeException** -> **SortedListTopNumIsNonNegativeException**